NAS2D/Renderer/Fade.cpp

```diff
@@ -29,15 +29,13 @@ SignalSource<>& Fade::fadeComplete()
 	return mFadeComplete;
 }
 
-
 // Fade in from fadeColor
 void Fade::fadeIn(unsigned int durationInMilliseconds)
 {
 	setDuration(durationInMilliseconds);
 	mDirection = FadeDirection::In;
 }
 
-
 // Fade out to fadeColor
 void Fade::fadeOut(unsigned int durationInMilliseconds)
```

Comment:
Inter-function spacing was changed here. That makes it inconsistent with the rest of the file.

Reply:
Fixed